tests/integration/targets/iosxr_ospfv3/tests/cli/parsed.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- debug:
+    msg: START iosxr_ospfv3 parsed integration tests on connection={{ ansible_connection
+      }}
+
+- block:
+
+    - name: Use parsed state to convert externally supplied device specific ospfv3
+        routes commands to structured format
+      register: result
+      cisco.iosxr.iosxr_ospfv3:
+        running_config: "{{ lookup('file', '../../fixtures/parsed.cfg') }}"
+        state: parsed
+
+    - assert:
+        that: "{{ merged['after'] == result['parsed'] }}"
```

Comment:
Can an assertion be added , for all non-action states, to check the value of result['changed'] ?